zen/scene/scene.c

```diff
@@ -51,6 +51,66 @@ zn_scene_move_board_binding_handler(
   }
 }
 
+static void
+zn_scene_switch_focused_view_binding_handler(
```

Comment:
We decided to use `handle_*` instead of `*_handler`.
https://github.com/zigen-project/zen/pull/157#issuecomment-1250365166
https://github.com/zigen-project/zen/commit/b9d2cc51553efc4466d9ccfd0fdd81ddd982f838

Could you use `handle_*` and fix `*_handler` functions in this file?

```suggestion
zn_scene_handle_switch_focused_view_binding(
```